invoke.example.yml

```
@@ -1,9 +1,9 @@
 ---
 nautobot_dev_example:
   project_name: "nautobot-dev-example"
-  nautobot_ver: "2.0.0"
+  nautobot_ver: "2.3.0"
```

```suggestion
  nautobot_ver: "2.3.1"
```
I think this will need to be 2.3.1 to avoid issues with pylint-nautobot once it's released.